pkg/namespaces/namespaces.go

```diff
@@ -102,6 +102,11 @@ func (n UsernsMode) IsAuto() bool {
 	return parts[0] == "auto"
 }
 
+// IsDefaultValue indicates whether the user namespace has the default value.
+func (n UsernsMode) IsDefaultValue() bool {
```

Comment:
Shouldn't this also check for the explicit Default constant?

Reply:
I've copied it from cgroupns.

I've fixed both and pushed a new version